appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -131,6 +133,10 @@ public class Appcues internal constructor(internal val scope: AppcuesScope) {
      * @param properties Optional properties that provide additional context about the user.
      */
     public fun identify(userId: String, properties: Map<String, Any>? = null) {
+        if (pushToken != null && pushToken != storage.pushToken) {
+            storage.pushToken = pushToken
+        }
```

Comment:
should it reset the `pushToken` value in companion to nil after this check?

Reply:
checking understanding, we don't call `setPushToken(...)` directly since the session_started event that is coming here will handle the device update, and the separate device_updated event is not needed in this case.